src/logger.c

```diff
@@ -40,59 +40,62 @@ static const char *logident = NULL;
 bool logcontrol = false;
 int umbilical = 0;
 
+static bool should_log(int level){
+	// Bail out early if there is nothing to do.
+
+	if(!logcontrol && (level > debug_level || logmode == LOGMODE_NULL)) {
+		return false;
+	}
+
+	return (level > debug_level);
```

Comment:
That should be `return level <= debug_level`.

Reply:
how did I miss this :(

fixed.